include/open62541pp/Server.h

```diff
@@ -147,6 +158,7 @@ class Server {
     ServerContext& getContext() noexcept;
 
 private:
+    void init(uint16_t port, ByteString certificate);
```

Comment:
I would like to avoid a new (private) member function. A simple free function should work (see below).